core/network/fabric-interop-cc/interfaces/asset-mgmt/asset_locks.go

```diff
@@ -136,7 +136,7 @@ func (am *AssetManagement) LockAsset(stub shim.ChaincodeStubInterface, assetAgre
     iccResp := stub.InvokeChaincode(am.interopChaincodeId, [][]byte{[]byte("LockAsset"), []byte(assetAgreementBytes64), []byte(lockInfoBytes64)}, "")
     fmt.Printf("Response from Interop CC: %+v\n", iccResp)
     if iccResp.GetStatus() != shim.OK {
-        return "", logThenErrorf(string(iccResp.GetPayload()))
+        return "", logThenErrorf(string(iccResp.Message))
```

Comment:
Returning payload instead of message is fine, but I wonder if the message may also contain something useful if the invocation didn't work (sometimes the failure occurs outside the chaincode function). Maybe return both in a JSON? Also, I think the message can be obtained using GetMessage().

Reply:
I have updated `iccResp.Message` with `iccResp.GetMessage()`. The Payload() can be very long as well, and I guess logging that is good enough so that the chaincode container log can be inspected in case of any errors.